verify_mapping_quality.pl

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use Pod::Usage;
+use Getopt::Long;
+use autodie;
+use FindBin qw($Bin);
+use File::Basename;
+use Parallel::ForkManager;
+use Readonly;
+
+Readonly my $MIN_DEPTH => 10;    # THIRD COLUMN
+Readonly my $MIN_MAP => 80;
+__PACKAGE__->run unless caller;
+
+1;
+
+sub run {
+    my ( $dir, $size, $man, $help, $min_depth, $log_dir, $min_map );
+
+    GetOptions(
+        "i|dir=s"     => \$dir,
```

Comment:
passing a directory will work when you run the script on the command line but it will not work well with Galaxy.  A good way to get it work on galaxy would be the following example in https://github.com/apetkau/vcf2pseudoalignment/blob/master/vcf2pseudoalignment.pl#L584 

So you will be able to write on the command line i.e :
--bam file1.bam --bam file2.bam --bam file3.bam
etc...

Document @ https://metacpan.org/pod/Getopt::Long#Options-with-hash-values


You should keep both ways otherwise whoever runs on the command line is going to hate you.... :)